docs/source/actions.rst

```diff
@@ -178,7 +178,7 @@ Slack action
 
 
 This action allows you to send custom messages to a `Slack`_ channel (or user)
-using a bot when your monitor suites finish their execution.
+using a bot when your monitor suites finishes their execution.
```

Comment:
It’s actually correct, “suites” is plural.

Reply:
True that, implemented.